.mega-linter.yml

```diff
@@ -27,6 +27,7 @@ DISABLE_LINTERS:
   - REPOSITORY_DEVSKIM
   - REPOSITORY_TRUFFLEHOG
   - REPOSITORY_GIT_DIFF
+  - MARKDOWN_MARKDOWN_LINK_CHECK
```

Comment:
Instead of globally ignoring this rule, we could add `<!-- markdown-link-check-disable-next-line -->` on the line above where the localhost link in the readme causes the failure? Just because this lint rule may be useful if we add links in future, and can be flexible with what we ignore?

https://github.com/ONSdigital/eq-cir-converter-service/pull/4/files#r1880178441

Reply:
Excellent point, we can just disable one line instead.